frontend/src/e2e/README.md

```diff
@@ -0,0 +1,41 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+### Writing test cases
+
+1. Install dependencies then run in development mode
+
+```sh
+$ yarn install && yarn start
```

Comment:
How about we use `makefile` to run test cases? In this way, developers only need to know the entry of `make`.

We can implement this separately later.

Reply:
> How about we use `makefile` to run test cases? In this way, developers only need to know the entry of `make`.
> 
> We can implement this separately later.

Good idea, but I'm not familiar with `makefile`  and need time to research it. If that's OK, I'll use it in the future.

Reply:
here is an example of APISIX, you can take a look: https://github.com/apache/apisix/blob/master/Makefile#L70